File: CentralIT API/CentralIT.Domain/Request/RequestFilter.cs
using System;
using System.Collections.Generic;
using System.Text;

using CentralIT.Domain.Users;

namespace CentralIT.Domain.Requests {
    public class RequestFilter {

        public int RequestId { get; set; }

        public int UserRequestedId { get; set; }

        public int UserAttendedId { get; set; }

        public int RequestTypeId { get; set; }

        public int PriorityId { get; set; }

        public int ServiceId { get; set; }

        public int DepartmentId { get; set; }

        public int Status { get; set; }

        public DateTime RequestDate { get; set; }

        public DateTime AttendedDate { get; set; }
    }
}

File: CentralIT/src/app/requests/shared/viewmodels/requestfilter.ts
import { UserModel } from "src/app/models/security/user";

export class RequestFiltesViewModel{
    request: string = "";
    requestId: number = -1;
    userRequested: string = "";
    userAttended: string = "";
    userRequestedId: number = -1;
    userAttendedId: number = -1;
    requestTypeId: number = -1;
    priorityId: number = -1;
    serviceId: number = -1;
    departmentId: number = -1;
    attendedDateS: string = "01/01/1900";
    requestDateS: string = "01/01/1900";
    status: number = -1;
}